infrastructure/src/main/java/org/corfudb/infrastructure/LayoutServer.java

```diff
@@ -138,6 +163,14 @@ public synchronized void handleMessageLayoutRequest(CorfuPayloadMsg<Long> msg, C
      */
     @ServerHandler(type=CorfuMsgType.LAYOUT_BOOTSTRAP)
     public synchronized void handleMessageLayoutBootstrap(CorfuPayloadMsg<LayoutBootstrapRequest> msg, ChannelHandlerContext ctx, IServerRouter r) {
+        Timer.Context context = timerLayoutBootstrap.time();
+        try {
```

Comment:
I'm not familiar enough with annotation processing to attempt it. I did search for such support in the dropwizard library but didn't find it.